include/pistache/typeid.h

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 #include <functional>
+#include <stddef.h>
```

Comment:
Can you change this to cstddef? Or is that part of the problem?

Reply:
If I remember correctly cstddef contains only std::size_t (by the standard), while the code uses size_t. In practice size_t is almost always included as well, but since Pistache is compiled and used on a variety of platforms with different stdlibs it should be better to follow the standard closely

Reply:
I think another option would be to err on the size of the standard and use cstddef, and then correct `size_t` to be prefixed with the `std` name space in any code that breaks later.